frontend/src/chall.tsx

```diff
@@ -260,11 +261,17 @@ const Chall = () => {
                             <Stop className="buttonsvg r" />
                         </button>
                     )}
-                    {ports.map((p: string | JSX.Element) => (
-                        <div className="IP-port-box" key={p as string}>
-                            {p}
+                    {Date.now() / 1000 > deployment.start_delay ? (
+                        ports.map((p: string | JSX.Element) => (
+                            <div className="IP-port-box" key={p as string}>
+                                {p}
+                            </div>
+                        ))
+                    ) : (
+                        <div className="IP-port-box" key="loading">
+                            loading challenge information...
```

Comment:
Maybe instead of "loading challenge information..." we should say something more accurate like "challenge is booting up" or something?

Reply:
yeah I agree, will fix